src/platform-includes/getting-started-install/unreal.mdx

```diff
@@ -1,4 +1,4 @@
-The Unreal Engine (UE) SDK is currently compatible with **UE 4.27 or later** and supports **Windows**, **Linux**, **macOS**, **iOS**, and **Android**.
+The Unreal Engine (UE) SDK is officially supported for the three latest UE versions. However, it is likely to be compatible with older engine versions as well depending on the specific features and functionality that you need.
```

Comment:
Instead of saying "the three latest UE versions" can we say "version X and above" and use a specific version number? This way, we won't need to go back and update the docs for supported versions as much.

Reply:
@vivianyentran The whole point here was to avoid mentioning any specific engine version explicitly. Supporting the three latest UE versions is a requirement set by Epic Games, so we can safely drop the "X" whenever the new engine release goes live. If I got this right having "version X and above" will require us to replace "X" with "X+1" every time engine gets updated.

Reply:
Ah, gotcha, that makes sense. Thanks for clarifying!